Specs/Sequence/English/IpAddressModel.json

```diff
@@ -0,0 +1,359 @@
+[
+{
+  "Input": "My PC IP address is 1.1.1.1",
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "1.1.1.1",
+      "TypeName": "ip",
+      "Resolution": {
+        "value": "1.1.1.1"
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "My PC IP address is 0.0.0.0",
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "0.0.0.0",
+      "TypeName": "ip",
```

Comment:
Should we have subtypes here? Or another attribute in resolution to say ipv4 or ipv6?

Reply:
Should we also support subnet masks? Like 216.3.128.128 (/25)

Reply:
We will support subnet masks in next pr